posthog/api/test/test_authentication.py

```diff
@@ -434,6 +434,23 @@ def test_cant_reset_more_than_six_times(self):
         # Three emails should be sent, fourth should not
         self.assertEqual(len(mail.outbox), 6)
 
+    def test_is_rate_limited_on_email_not_ip(self):
```

Comment:
i'm lazily assuming there's a test in here that shows the reverse (if we have the same IP but varying addresses then we don't limit (or we're slower to limit))